### PR TITLE
Add unreferenced support libraries to cql_dependency_structure.

### PR DESCRIPTION
**Pull request by jbradl11**
Pull requests into Bonnie Bundler require the following. Submitter and reviewer should :white_check_mark: when done. For items that are not-applicable, note it's not-applicable ("N/A") and :white_check_mark:.

A measure will fail to load properly if a measure has a support library, but does not use any of the definitions or functions in that library and instead only references a value set or a DRC from the library. This issue is due how we build cql_dependency_structure. In order to fix this, we now add all libraries to the cql_dependency_structure. If the library has no dependencies we assign it to an empty hash. 

**Submitter:**
- [x] This pull request describes why these changes were made.
- [x] This PR is into the correct branch.
- [x] JIRA ticket for this PR: https://jira.mitre.org/browse/BONNIE-1270
- [x] JIRA ticket links to this PR: 
- [x] Code diff has been done and been reviewed (it **does not** contain: additional white space, not applicable code changes, debug statements, etc.)
- [x] Tests are included and test edge cases
- [x] Tests have been run locally and pass (remember to update Gemfile when applicable)
- [x] Code coverage has not gone down and all code touched or added is covered.
    * In rare situations, this may not be possible or applicable to a PR. In those situations:
        1. Note why this could not be done or is not applicable here:
        2. Add TODOs in the code noting that it requires a test
        3. Add a JIRA task to add the test and link it here:
- [x] Automated regression test(s) pass

If JIRA tests were used to supplement or replace automated tests:
- [x] JIRA test links: N/A
- [x] Justification for using JIRA tests: N/A
- [x] JIRA tests have been added to sprint: N/A

**Reviewer 1:**

Name: @losborne
- [x] Code is maintainable and reusable, reuses existing code and infrastructure where appropriate, and accomplishes the task’s purpose
- [x] The tests appropriately test the new code, including edge cases
- [x] You have tried to break the code

If JIRA tests were used to supplement or replace automated tests:
- [x] JIRA tests have been run and pass N/A
- [x] You agree with the justification for use of JIRA tests or have provided input on why you disagree N/A

**Reviewer 2:** @mayerm94

Name:
- [x] Code is maintainable and reusable, reuses existing code and infrastructure where appropriate, and accomplishes the task’s purpose
- [x] The tests appropriately test the new code, including edge cases
- [x] You have tried to break the code

If JIRA tests were used to supplement or replace automated tests:
- [x] JIRA tests have been run and pass
- [x] You agree with the justification for use of JIRA tests or have provided input on why you disagree

### CODE REVIEW

File: test/unit/cql_loader_test.rb [hunk 1]
```diff
@@ -52,6 +52,26 @@ class CQLLoaderTest < ActiveSupport::TestCase
...
+      dump_db
+      user = User.new
+      user.save
+
+      measure_details = { 'episode_of_care'=> false }
+      Measures::CqlLoader.load(unused_library_mat_export, user, measure_details, ENV['VSAC_USERNAME'], ENV['VSAC_PASSWORD'], false, false, true, nil).save
+      assert_equal 1, CqlMeasure.all.count
+      measure = CqlMeasure.all.first
+
+      # Confirm that the source data criteria with the direct reference code is equal to the expected hash
```
Comment (lukoktonos) [inline test/unit/cql_loader_test.rb]: This comment doesn't seem to match the following line.

File: test/unit/cql_loader_test.rb [hunk 2]
```diff
@@ -52,6 +52,26 @@ class CQLLoaderTest < ActiveSupport::TestCase
...
 
+
+  test 'Loading a measure with support libraries that dont have their define definitions used are still included in the dependencty structure as empty hashes' do
+    unused_library_mat_export = File.new File.join('test', 'fixtures', 'PVC2_v5_4_Unused_Support_Libraries.zip')
+    VCR.use_cassette('valid_vsac_response_pvc_unused_libraries') do
+      dump_db
+      user = User.new
+      user.save
+
+      measure_details = { 'episode_of_care'=> false }
```
Comment (lukoktonos) [inline test/unit/cql_loader_test.rb]: There's a missing space before `=>`

**Review by lukoktonos — CHANGES_REQUESTED** [review]

**Review by lukoktonos — APPROVED** [review]
Looks good

**Review by mayerm94 — APPROVED** [review]

**Comment by edeyoung** [comment]
failing due to security vulnerabilities - need to update to latest prod release
